tune_sklearn/utils.py

```diff
@@ -1,6 +1,16 @@
 from sklearn.metrics import check_scoring
 from sklearn.pipeline import Pipeline
+from tune_sklearn._detect_xgboost import is_xgboost_model
 import numpy as np
+from enum import Enum
+
+
+class EarlyStopping(Enum):
+    PARTIAL_FIT = 1
+    WARM_START_ITER = 2
+    WARM_START_ENSEMBLE = 3
+    XGB = 4
+    NO_EARLY_STOP = 5
```

Comment:
prefer string value instead of numeric (for better readability when serialized/debugged)